.idea/gradle.xml

```diff
@@ -14,13 +14,7 @@
             <option value="$PROJECT_DIR$/example_app" />
           </set>
```

Comment:
`.gitignore` should probably be updated to include all these other automatically re-generated Android Studio files.